autogluon/utils/tabular/ml/models/tabular_nn/tabular_nn_model.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, path: str, name: str, problem_type: str, objective_func, stop
         self.ctx = mx.cpu()
         self.batch_size = None
         self.num_dataloading_workers = None
+        self.num_dataloading_workers_inference = 0
```

Comment:
In the future, should we make this a hyperparameter of the NN model instead?

Reply:
I agree that it could be, if you'd like to make it a hyperparameter we can do so in future.